modules/saliency/src/staticSaliencyFineGrained.cpp

```diff
@@ -66,7 +66,9 @@ bool StaticSaliencyFineGrained::computeSaliencyImpl(InputArray image, OutputArra
 {
     Mat dst(Size(image.getMat().cols, image.getMat().rows), CV_8UC1);
     calcIntensityChannel(image.getMat(), dst);
-    dst.copyTo(saliencyMap);
+    double minVal, maxVal;
+    minMaxLoc( dst, &minVal, &maxVal );
+    dst.convertTo(saliencyMap, CV_32F, 1.0f/maxVal);
```

Comment:
`minMaxLoc` approach doesn't look reliable here.

dst type is `CV_8UC1`, so just apply 1/255.0f scale:
```
dst.convertTo(saliencyMap, CV_32F, 1.0f/255.0f);  // values are in range [0; 1]
```